cogs5e/sheetManager.py

```diff
@@ -385,6 +386,34 @@ async def character(self, ctx, *, name: str = None):
 
         await ctx.send(f"Active character changed to {char.name}.", delete_after=20)
 
+    @character.command(name='server')
+    @commands.guild_only()
+    async def character_server(self, ctx, *, name: str = None):
+        """Toggles the active character on the current server."""
+        char: Character = await Character.from_ctx(ctx)
+        
+        if char.is_active_server(ctx):
+            await char.set_server_active(ctx)
+            msg = f"Active server character set to {char.name}."
+        else:
+            await char.unset_server_active(ctx)
+            msg = f"Active server character unset from {char.name}."
+        
+        await try_delete(ctx.message)
+        
+        await ctx.send(msg, delete_after=20)
+
+    @character.command(name='global')
+    async def character_global(self, ctx):
+        """Shows the active global character."""
+        user_characters = await self.bot.mdb.characters.find({"owner": str(ctx.author.id)}).to_list(None)
+        if not user_characters:
+            return await ctx.send('You have no characters.')
+        
+        active_character: Character = await Character.from_ctx(ctx,ignoreGlobal=True)
+        embed = active_character.active_embed()
+        await ctx.send(embed=embed)
```

Comment:
I think this command can be removed and the information about a global character can be added to the active embed.

Reply:
So you want the active embed to show information for both?